Description: Include sched.h with GNU extensions
 This hopefully fixes FTBFS on IA64 (but oddly not on other archs):
 https://buildd.debian.org/status/fetch.php?pkg=uwsgi&arch=ia64&ver=1.9.17.1-2&stamp=1380102648
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2013-09-24

--- a/uwsgi.h
+++ b/uwsgi.h
@@ -152,7 +152,7 @@
 #define _GNU_SOURCE
 #endif
 #include <stdio.h>
-#ifdef __UCLIBC__
+#if defined(__linux) || defined(__UCLIBC__)
 #include <sched.h>
 #endif
 #undef _GNU_SOURCE
@@ -236,7 +236,6 @@
 
 
 #ifdef __linux__
-#include <sched.h>
 #include <sys/prctl.h>
 #include <linux/limits.h>
 #endif
